doc/asab/config.rst

```diff
@@ -214,3 +214,67 @@ The obtainment of the second value in the code can be achieved in two ways:
 	self.SleepTime = asab.Config["sleep"].getseconds("sleep_time")
 	self.AnotherSleepTime = asab.Config.getseconds("sleep", "another_sleep_time")
 
+Obtaining URLs
```

Comment:
Nice fairytale :)
Use typing.
I think three lines should be enough to understand how to use the method. Plus example of happy flow.

Reply:
I enjoyed my fairytale however I understand. Shortened it